src/services/Onepassword.ts

```diff
@@ -15,9 +15,14 @@ import {
 import { device } from "../config";
 
 export class Onepassword {
-  private device: Device = device;
+  private device: Device;
   private session: Session;
-  private requestService: RequestService = new RequestService();
+  private requestService: RequestService;
+
+  constructor(userDevice: Device = device) {
```

Comment:
Would it make sense to merge what the user passed with the default device instead, as in most occasions you will just override the client version

Reply:
updated